lego/apps/users/models.py

```diff
@@ -160,7 +160,8 @@ def is_superuser(self):
 
     @property
     def is_abakus_member(self):
-        return 'Abakus' in [group.name for group in self.all_groups]
+        return constants.MEMBER_GROUP in [group.name for group in self.all_groups]
```

Comment:
👍 

not that it's relevant to this PR, but this seems a bit inefficient in general. do we have to retrieve all groups to figure out if one of them is abakus? maybe there's a more efficient way to figure that out with MPTT. 

if the reasoning is that `self.all_groups` is a cached property and this always results in a cache hit, we should at least use a regular for loop to avoid having to store the whole list of group names in memory (and then going through it again to do the `in` lookup): 

```python
for group in self.all_groups:
    if group.name == constants.MEMBER_GROUP:
        return True

return False
```